components/src/organisms/quantity-update/QuantityUpdate.html

```diff
@@ -0,0 +1,70 @@
+<div class="a-quantity-update">
+  <!-- @slot Label (Named slot) -->
+  <slot name="label">
+    <a-label
+      v-if="label"
+      class="a-quantity-update__label"
+      :for="inputId"
+    >
+      {{ label }}
+    </a-label>
+  </slot>
+  <div class="a-quantity-update__qty">
+    <!-- @slot Decrement qty button (Scoped slot) -->
+    <slot
+      name="minus"
+      :update-qty="updateQty"
+    >
+      <a-button
+        class="a-quantity-update__button a-quantity-update__button--minus"
+        :disabled="currentValue === minQty"
+        :aria-label="decrementAriaLabel"
+        @click.native="updateQty(-1)"
+      >
+        <a-icon
+          icon="minus"
+          class="button__icon a-quantity-update__icon"
+        >{{ decrementIconTitle }}</a-icon>
+      </a-button>
+    </slot>
+    <!-- @slot Qty input (Scoped slot) -->
+    <slot
+      name="input"
+      :current-value="currentValue"
+      :input-event="inputEvent"
+    >
+      <a-input-wrapper
+        :input-id="inputId"
+        class="a-quantity-update__input-wrapper"
+      >
+        <input
+          :id="inputId"
+          class="a-quantity-update__input"
+          type="number"
+          :min="minQty"
+          :max="maxQty"
+          :value="currentValue"
+          @input="inputEvent"
+          autocomplete="off"
+        />
+      </a-input-wrapper>
+    </slot>
+    <!-- @slot Increment qty button (Scoped slot) -->
+    <slot
+      name="plus"
+      :update-qty="updateQty"
+    >
+      <a-button
+        class="a-quantity-update__button a-quantity-update__button--plus"
+        :disabled="currentValue === maxQty"
+        :aria-label="incrementAriaLabel"
+        @click.native="updateQty(1)"
+      >
+        <a-icon
+          icon="plus"
+          class="a-quantity-update__icon"
+        >{{ incrementIconTitle }}</a-icon>
```

Comment:
I change in `Icon` component title from `<slot/>` to `title` prop and there we have
```
<title>{{ titleComputed }}</title>
```
it must be in one line to avoid problem with multiline title tooltip

Reply:
ok